R/row_to_names.R

```diff
@@ -22,16 +22,14 @@
 #' x %>%
 #'   row_to_names(row_number = "find_header")
 #' @export
-row_to_names <- function(dat, row_number, ..., remove_row = TRUE, remove_rows_above = TRUE) {
+row_to_names <- function(dat, row_number, ..., remove_row = TRUE, remove_rows_above = TRUE, sep = "_") {
   # Check inputs
   if (!(is.logical(remove_row) & length(remove_row) == 1)) {
     stop("remove_row must be either TRUE or FALSE, not ", as.character(remove_row))
   } else if (!(is.logical(remove_rows_above) & length(remove_rows_above) == 1)) {
     stop("remove_rows_above must be either TRUE or FALSE, not ", as.character(remove_rows_above))
-  } else if (length(row_number) != 1) {
-    stop("row_number must be a scalar")
   }
-  if (row_number %in% "find_header") {
+  if (any(row_number %in% "find_header")) {
```

Comment:
This would be better as `identical(row_number, "find_header")` because having some rows as numbers and one as `"find_header"` would not be an input we would want to support.

Reply:
I agree, it was changed.